docs/source/cli.mdx

```diff
@@ -0,0 +1,48 @@
+# Command Line Interface (CLI)
+
+🤗 Datasets provides a command line interface (CLI) with useful shell commands to interact with your dataset.
+
+You can check the available commands:
+```shell
+$ datasets-cli --help
+usage: datasets-cli <command> [<args>]
+
+positional arguments:
+  {convert,env,test,run_beam,dummy_data,convert_to_parquet}
+                        datasets-cli command helpers
+    convert             Convert a TensorFlow Datasets dataset to a HuggingFace Datasets dataset.
+    env                 Print relevant system environment info.
+    test                Test dataset implementation.
+    run_beam            Run a Beam dataset processing pipeline
+    dummy_data          Generate dummy data.
+    convert_to_parquet  Convert dataset to Parquet
+
+optional arguments:
+  -h, --help            show this help message and exit
+```
+
+## Convert to Parquet
+
+Easily convert your Hub script-dataset to Parquet files, so that the dataset viewer will be supported.
```

Comment:
```suggestion
Easily convert your Hub [script-based dataset](https://huggingface.co/docs/datasets/dataset_script) to [data-only](https://huggingface.co/docs/hub/datasets-manual-configuration), so that the dataset viewer will be supported.
```